ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactView)
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let labelMinimumHeight: CGFloat = 16.0
+        static let titleLabelMaximumHeight: CGFloat = 28.0
+        static let subtitleMaximumHeight: CGFloat = 24.0
+        static let spacingBetweenAvatarAndLabelContainer: CGFloat = 13.0
+    }
+
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            if let subtitleLabel = subtitleLabel {
+                setupAvatarView(with: titleLabel.text ?? "", and: subtitleLabel.text ?? "")
+            } else {
+                setupAvatarView(with: titleLabel.text ?? "")
+            }
+        }
+    }
+
+    private let avatarView: AvatarView
+    private var titleLabel: UILabel
+    private var subtitleLabel: UILabel?
+    private var labelContainer: UIView
+
+    /// Initializes the contact view by creating an avatar view with a first name and last name
```

Comment:
>first name and last name [](start = 71, length = 24)

I would word this as something like "primary and secondary text"

Reply:
Yes, of course. I must have missed this since the refactoring tool doesn't change the comments.

Reply:
I think we should generalize it

Reply:
The reason why I think this is very important is because I'm not sure clients will always be able to make the distinction between a name and last name, that structure and it's ordering I'm not sure if it's global and applies for every language.

Reply:
Got it, thanks for explaining!